core/domain/classifier_services.py

```diff
@@ -118,7 +118,19 @@ class StringClassifier(object):
 
     _DEFAULT_PREDICTION_THRESHOLD = 0.5
 
-    _DEFAULT_LABEL = '_default'
+    """Classifiers built with less than _DEFAULT_MIN_DOCS_TO_PREDICT will
+    likely not be useful for predicting as there are not enough examples to
+    build a generalized model.
+    """
+    _DEFAULT_MIN_DOCS_TO_PREDICT = 20
+    """Because prediction uses Prob(the doc should be assigned this label | the
+    doc is not assigned DEFAULT_LABEL), if there are only two labels (the
+    default label and one other) then the one other label will always be
+    predicted.
```

Comment:
I think you should end with a final thought which ties the reasoning explained in the docstring with the value '3', such as:

*Thus, there needs to be at least 3 labels in order to perform valid prediction.*